src/figuresClasses.ts

```diff
@@ -2,10 +1,76 @@
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape = 'triangle';
+
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Values must be greater than 0');
+    }
+
+    if (a + b <= c || a + c <= b || b + c <= a) {
+      throw new Error('Sides do not form a valid triangle');
+    }
+  }
+
+  getArea(): number {
+    const s = (this.a + this.b + this.c) / 2;
+    const area = Math.sqrt(s * (s - this.a) * (s - this.b) * (s - this.c));
+
+    return Number(area.toFixed(2));
+  }
+}
+
+export class Circle implements Figure {
+  shape = 'circle';
 
-export class Triangle implements Figure {}
+  constructor(
+    public color: string,
+    public radius: number,
+  ) {
+    if (radius <= 0) {
```

Comment:
The constructor uses the parameter `radius` directly, but it should be accessed using `this.radius` since it is a class property. This will cause a reference error.